### ISSUE TITLE
DLC problem in Fire Emblem Fates Special Edition

### ISSUE THREAD

**Issue by Tsuhirito** [citra-emu/citra#4746]
From my New 2DS XL I can play every DLC for Fire Emblem Fates but when I install DLC CIA from CITRA I can see and play some DLC… The others are hidden. 

System Information

Operating System: Windows 10
CPU: i7 6700k
GPU: GTX 970
Citra Version (found in title bar): Citra Canary 1298
Game: Fire Emblem Fates Special Edition EUR

Screenshots:

![1](https://user-images.githubusercontent.com/29732138/56419340-2cf12e80-629a-11e9-9ce3-50f27ad98abe.JPG)

![2](https://user-images.githubusercontent.com/29732138/56419343-2f538880-629a-11e9-879d-230c2bd68c8a.jpg)

**Comment by SteveCraftwood** [comment]
Yeah same problem here, only a handful of the DLC is active despite it all being installed.

**Comment by dawnbomb** [comment]
does this same problem happen in awakening?

**Comment by SteveCraftwood** [comment]
> 
> 
> does this same problem happen in awakening?

Not to my knowledge, Awakening works pretty perfectly compared to some of Fates weird quirks.
